src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,24 +14,30 @@
     private Phone phone;
     private Email email;
     private Address address;
+    private int sequenceNumber;
 
     private final UniqueTagList tags;
+    private static int nextSequenceNumber = 0;
```

Comment:
Since you call it `nextSequenceNumber`, it would be better to initialize as 1....